src/composables/useGivingFund.js

```diff
@@ -23,6 +24,21 @@ export default function useGivingFund(apollo) {
 		return offsets;
 	};
 
+	/**
+	 * Get a users Giving Funds
+	 */
+	const fetchMyGivingFundsData = async () => {
+		try {
+			const response = await apollo.query({
+				query: myGivingFundsQuery,
```

Comment:
@eddieferrer I think we should just do a small inline query instead of using this existing one seeing as we only need count at this time. This current query pulls a lot of data from several services so it's pretty expensive.